Sandfire_Smothered_Magic/scripts/Inventory.gd

```diff
@@ -33,14 +33,16 @@ func slot_gui_input(event: InputEvent, slot: SlotClass):
 					if holding_item.spell_name != slot.spell.spell_name:
 						var temp_item = slot.spell
 						slot.pickFromSlot()
-						temp_item.global_position = event.global_position
+						#offset = event.global_position - slot.rect_global_position
```

Comment:
Uncomment this for slightly different functionality on swap. I like it better without this line, personally.

Reply:
Thank you, good idea.